mongo/errors.go

```diff
@@ -106,7 +106,7 @@ func IsTimeout(err error) bool {
 			return ne.Timeout()
 		}
 		//timeout error labels
-		if se, ok := err.(ServerError); ok {
+		if se, ok := err.(labeledError); ok {
```

Comment:
[nit] `le` or `e` instead of `se` since it's now `labeledError`.

Reply:
done